operations_acl_rules.tf

```diff
@@ -145,7 +145,7 @@ resource "aws_network_acl_rule" "operations_ingress_from_anywhere_via_ports_1024
   network_acl_id = aws_network_acl.operations.id
   egress         = false
   protocol       = each.value
-  rule_number    = 160 + index(local.tcp_and_udp, each.value)
+  rule_number    = 170 + index(local.tcp_and_udp, each.value)
```

Comment:
Why did this rule number (and the one just below this) need to change?

Reply:
Good eye- this was done because in the course of testing the Egress Assess instance, the previous NACL rule group (`aws_network_acl_rule.operations_ingress_from_anywhere_via_allowed_ports`) required more than 10 NACL rules (as specified by `var.inbound_ports_allowed["egressassess"]` in our test `tfvars` file) and that broke our old rule numbering scheme.  

To remedy this, we bumped up the subsequent rule numbers by 10 to avoid any conflicts.  We are now able to support use cases of up to 20 `operations_ingress_from_anywhere_via_allowed_ports` NACL rules instead of 10.

Reply:
Makes sense.